Source/WTF/wtf/PlatformHave.h

```diff
@@ -1655,6 +1655,10 @@
 #define HAVE_UI_ASYNC_DRAG_INTERACTION 1
 #endif
 
+#if __has_include(<UIKit/UIAsyncTextInput.h>) && __has_include(<UIKit/UIKeyEventContext.h>)
+#define HAVE_UI_ASYNC_TEXT_INPUT 1
```

Comment:
Indeed, it's _more like_ `HAVE(UI_ASYNC_TEXT_INPUT_AND_RELATED_CLASSES_AND_PROTOCOLS)`, where `RELATED_CLASSES` refers to a number of new interfaces. Instead of enumerating all the classes in the macro, I decided to just go with the main protocol, `UIAsyncTextInput` (or at least, the protocol with the majority of the functionality).